packages/medusa/src/types/price-list.ts

```diff
@@ -0,0 +1,151 @@
+import { Type } from "class-transformer"
+import {
+  IsEnum,
+  IsInt,
+  IsOptional,
+  IsString,
+  Validate,
+  ValidateIf,
+  ValidateNested,
+} from "class-validator"
+import { PriceList } from ".."
+import { DateComparisonOperator } from "./common"
+import { XorConstraint } from "./validators/xor"
+
+export enum PriceListType {
```

Comment:
Should these not be prefixed with either `Store` or `Admin`?

Reply:
The ones that are used in the API layer definitely should, I've prefixed those with `Admin*` now. 😄